src/main/resources/static/_shared.yml

```diff
@@ -2301,6 +2303,12 @@ components:
         - residencyManagement
         - bailAssessment
         - bailSentence
+    Cas1ApplicationNoticeType:
```

Comment:
just wondering if we could be a bit clearer with the terminology here? Would:

- `Cas1ApplicationTimelinessCategory` or
- `Cas1ApplicationTimelinessRating`

be clearer? 

It wasn't immediately clear to me what "notice" would represent here.

Reply:
👍